fastlane/metadata/i18ncli.js

```diff
@@ -0,0 +1,320 @@
+/* eslint no-console: 0 */
+
+const yargs = require( "yargs" );
+const fsp = require( "fs/promises" );
+const path = require( "path" );
+
+const SUPPORTED_IOS_METADATA_LOCALES = [
+  "ar-SA",
+  "ca",
+  "cs",
+  "da",
+  "de-DE",
+  "el",
+  "en-AU",
+  "en-CA",
+  "en-GB",
+  "en-US",
+  "es-ES",
+  "es-MX",
+  "fi",
+  "fr-CA",
+  "fr-FR",
+  "he",
+  "hi",
+  "hr",
+  "hu",
+  "id",
+  "it",
+  "ja",
+  "ko",
+  "ms",
+  "nl-NL",
+  "no",
+  "pl",
+  "pt-BR",
+  "pt-PT",
+  "ro",
+  "ru",
+  "sk",
+  "sv",
+  "th",
+  "tr",
+  "uk",
+  "vi",
+  "zh-Hans",
+  "zh-Hant",
+  "appleTV",
+  "iMessage",
+  "default"
+];
```

Comment:
These are locale codes that the App Store understands, no matter what operating system the software you're distributing runs on, right? So these are more like SUPPORTED_APP_STORE_LOCALES.

Also, `appleTV`, `iMessage`, and `default` are not locales, are they?

Reply:
Have renamed the const and removed `appleTV` and `iMessage`.
`default` is a locale I would say. It is the `default` locale to be used if the submitted app is not translated.